test/integration/customUrlRest.spec.ts

```diff
@@ -0,0 +1,20 @@
+import CustomUrlRestClient, { CustomUrlRestClientRestClient } from "./generated/customUrlRest/src";
+import { assert } from 'chai';
+
+describe("CustomRest Endpoint", () => {
+  let client: CustomUrlRestClientRestClient;
+  let clientOptions: any;
+  beforeEach(() => {
+    clientOptions = { allowInsecureConnection: true };
+    client = CustomUrlRestClient("host:3000", clientOptions);
+  });
+
+  it("should return 200", async () => {
```

Comment:
Can you also add a test using the shortcut methods?

Reply:
added